sycl/include/sycl/handler.hpp

```diff
@@ -362,6 +362,32 @@ class __SYCL_EXPORT handler {
                                 PI_ERROR_INVALID_OPERATION);
   }
 
+  constexpr static int AccessTargetMask = 0x7ff;
```

Comment:
I remember part of the post-commit failures were that this is considered an exported symbol on Windows. You should add `?AccessTargetMask@handler@_V1@sycl@@0HB` to sycl/test/abi/sycl_symbols_windows.dump and add a comment `/testwin` to trigger pre-commit Windows testing.